.github/workflows/ci.yml

```diff
@@ -70,35 +70,6 @@ jobs:
           TIMEOUT: ${{ env.TIMEOUT }}
       - run: ./scripts/run_task.sh coverage
 
-  e2e_precompile:
```

Comment:
There are no more remaining e2e precompile tests (`warp` has its own job)

Reply:
CI will fail if this job is not removed -- there's nothing left for it to run, so the job fails. (you can see that in previous runs of CI on this PR). I could leave the old code side by side so the job has something to run, and will pass but that seems sort of silly.

I still plan on making a PR to remove all the remaining deprecated NPM stuff once the warp precompile PR is up.